xbmc/addons/AddonManager.cpp

```diff
@@ -394,11 +394,11 @@ std::vector<std::string> CAddonMgr::MigrateAddons()
   std::vector<AddonInfoPtr> incompatible;
   GetIncompatibleAddonInfos(incompatible, true);
 
-  return DisableIncompatibleAddons(incompatible);
+  return DisableIncompatibleAddons(incompatible, false);
 }
 
 std::vector<std::string> CAddonMgr::DisableIncompatibleAddons(
-    const std::vector<AddonInfoPtr>& incompatible)
+    const std::vector<AddonInfoPtr>& incompatible, bool includedependencies)
```

Comment:
See other comment. I would not do this here at all instead having a separate utility function. This infers a meaning to what this function does whereas in fact all it does is filter the returned vector.